src/SDK/SmartDetectors.Extensions/AlertExtensions.cs

```diff
@@ -140,65 +133,6 @@ public static ContractsAlert CreateContractsAlert(this Alert alert, SmartDetecto
             return predicates;
         }
 
-#pragma warning disable CS0612 // Type or member is obsolete; Task to remove obsolete code #1312924
-
-        /// <summary>
-        /// Creates an <see cref="AlertPropertyLegacy"/> based on an alert presentation V1 property
-        /// </summary>
-        /// <param name="alert">The alert</param>
-        /// <param name="presentationAttribute">The attribute defining the presentation V1 of the alert property</param>
-        /// <param name="queryRunInfo">The query run information</param>
-        /// <param name="propertyStringValue">The property string value</param>
-        /// <returns>An <see cref="AlertPropertyLegacy"/></returns>
-        private static AlertPropertyLegacy CreateAlertPropertyLegacy(Alert alert, AlertPresentationPropertyAttribute presentationAttribute, QueryRunInfo queryRunInfo, string propertyStringValue)
```

Comment:
>QueryRunInfo [](start = 140, length = 12)

remove this class as well

Reply:
As we discussed, I'll remove this class after publishing the new NuGet

---
In reply to: [242828623](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/35#discussion_r242828623) [](ancestors = 242828623)

Reply:
why?

---
In reply to: [244528099](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/35#discussion_r244528099) [](ancestors = 244528099,242828623)

Reply:
@ said it's not in use anymore

---
In reply to: [244558940](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/35#discussion_r244558940) [](ancestors = 244558940,244528099,242828623)

Reply:
This class (QueryRunInfo) is part of a different repository. I'll publish a new NuGet of the monitoring appliance after this PR and then I'll update the NuGet in the other repository and remove the QueryRunInfo class from that repository.

---
In reply to: [244566473](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/35#discussion_r244566473) [](ancestors = 244566473,244565145,244558940,244528099,242828623)